FNML/overview.md

```diff
@@ -6,12 +6,18 @@ we combine <a>RML</a> with declarative function descriptions in <a>FnO</a>.
 <a>FnO</a> consists of -- among others -- <a>function descriptions</a> and <a>execution</a> descriptions.
 We describe these <a>executions</a> -- which link to specific <a>function descriptions</a> -- within the <a>RML mapping</a>.
 
-The <a>RML processor</a> then needs to interpret these <a>executions</a> correctly to know which values to assign to the parameters of the function.
-After executing the correct function, the <a>RML processor</a> SHOULD return the resulting value, to be used for further processing by the <a>RML processor</a>.
+A [=triples map=] generates triples from input data.
+An [=RML mapping=] uses [=output triples map=]s to generate output triples from input data.
+We use an intermediate [=function triples map=] to generate [=execution triples=] from input data,
```

Comment:
but these execution triples are the ones that cause confusion and we might need to better define them.
I'd suggest that we try to add a clear definition